src/app/(auth)/layout.tsx

```diff
@@ -3,6 +3,9 @@ import AuthTopBar from './AuthTopBar';
 import { createServerComponentClient } from '@supabase/auth-helpers-nextjs';
 import { cookies } from 'next/headers';
 
+
+//todo wygląda jak kopia z docsów, ale jakbyś chciał prostsze typowanie childrenów to coś takiego próbuj const AuthLayout: FunctionComponent<PropsWithChildren> = ({children}) => {}
```

Comment:
Done, jak coś na razie wprowadzam zmiany lokalnie, chcę wrzucić wszystko jak skończę, a gdybym się nie wyrobił przed naszym 1on1 to chciałbym mieć jakieś notatki z tego :)